js/about/certerror.js

```diff
@@ -50,8 +47,8 @@ class CertErrorPage extends React.Component {
       </svg>
       <div className='certErrorText'>
         <span data-l10n-id='certErrorText'></span>&nbsp;
-        <span className='errorUrl'>{this.state.certDetails.get('url') || ''}</span>
-        <span className='errorText'>{this.state.certDetails.get('error') || ''}</span>
+        <span className='errorUrl'>{this.state.url || ''}</span>
+        <span className='errorText'>{this.state.error || ''}</span>
```

Comment:
I noticed that too, but I didn't make any changes it so I'd leave that with @bradleyrichter